modules/migration-kubernetes-objects.adoc

```diff
@@ -5,9 +5,16 @@
 
 :_content-type: PROCEDURE
 [id="migration-kubernetes-objects_{context}"]
-= Migrating Kubernetes objects
+= Migrating Kubernetes objects using the {mtc-short} API
 
-You can perform a one-time migration of Kubernetes objects that constitute an application's state.
+After you migrate all the PV data, you can the {mtc-full} ({mtc-short}) API to perform a one-time migration of Kubernetes objects that constitute an application's state.
+
```

Comment:
@RichardHoch 
"...you can use the {mtc-full}..."

Reply:
I do not understand your comment. Are you saying that it's wrong to include the abbreviation after the full name of the product?

Reply:
@RichardHoch 
No, I am saying that the word 'use' is missing!

Reply:
right.